### ISSUE TITLE
Homepage Accessibility Faliures caused by the blue color on book panels and :hover css

### ISSUE THREAD

**Issue by eshellman** [gutenbergtools/gutenbergsite#213]
<div class="offcanvas-header" style="box-sizing: border-box; position: relative; align-items: flex-start; display: flex; justify-content: space-between; padding: 35px 35px 0px; color: rgb(0, 0, 0); font-family: Rubik, &quot;Noto Sans Meetei Mayek&quot;, sans-serif; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="w-100" style="box-sizing: border-box; position: relative; width: 1002.7px;"><div style="box-sizing: border-box; position: relative;"><h2 class="h1 offcanvas-title" style="box-sizing: border-box; position: relative; font-weight: 600; line-height: 1.5; margin-bottom: 0px; margin-top: 0px; font-size: 2rem;">A contrast ratio of at least 3:1 should be provided.</h2><p style="box-sizing: border-box; position: relative; margin-bottom: 1rem; margin-top: 0px;">All text elements must have sufficient contrast between text in the foreground and background colors behind it in accordance with WCAG 2 AA contrast ratio thresholds.</p></div></div><button type="button" class="btn-close" aria-label="Close" style="box-sizing: initial; position: relative; border-radius: 0.375rem; font-family: inherit; font-size: inherit; line-height: inherit; margin: -17.5px -17.5px -17.5px 0px; text-transform: none; appearance: button; background: url(&quot;data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'%3E%3Cpath d='M.293.293a1 1 0 0 1 1.414 0L8 6.586 14.293.293a1 1 0 1 1 1.414 1.414L9.414 8l6.293 6.293a1 1 0 0 1-1.414 1.414L8 9.414l-6.293 6.293a1 1 0 0 1-1.414-1.414L6.586 8 .293 1.707a1 1 0 0 1 0-1.414z'/%3E%3C/svg%3E&quot;) 50% center / 1em no-repeat rgba(0, 0, 0, 0); border: 0px; color: rgb(0, 0, 0); height: 1em; opacity: 0.5; padding: 17.5px; width: 1em; cursor: pointer;"></button></div><div class="offcanvas-body" style="box-sizing: border-box; position: relative; flex-grow: 1; overflow-y: auto; padding: 1rem 35px 35px; color: rgb(0, 0, 0); font-family: Rubik, &quot;Noto Sans Meetei Mayek&quot;, sans-serif; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="violation-meta-wrapper" style="box-sizing: border-box; position: relative;"><div class="disabilities-affected" style="box-sizing: border-box; position: relative; margin-top: 0px;"><span role="heading" class="header" style="box-sizing: border-box; position: relative; font-weight: 700;">Disabilities Affected</span><ul class="disabilities-list" style="box-sizing: border-box; position: relative; padding: 0px; margin-bottom: 0px; margin-top: 0px; display: flex; flex-wrap: wrap; list-style-type: none;"><li class="visually_impaired" style="box-sizing: border-box; position: relative; align-items: center; display: flex; background-color: rgb(255, 255, 255); border: 2px solid rgb(74, 102, 129); border-radius: 5px; font-size: 1rem; margin-inline-end: 10px; margin-bottom: 10px; padding: 5px 15px;">Visually Impaired</li><li class="color_blind" style="box-sizing: border-box; position: relative; align-items: center; display: flex; background-color: rgb(255, 255, 255); border: 2px solid rgb(74, 102, 129); border-radius: 5px; font-size: 1rem; margin-inline-end: 10px; margin-bottom: 10px; padding: 5px 15px;">Color Blind</li><li class="dyslexia" style="box-sizing: border-box; position: relative; align-items: center; display: flex; background-color: rgb(255, 255, 255); border: 2px solid rgb(74, 102, 129); border-radius: 5px; font-size: 1rem; margin-inline-end: 10px; margin-bottom: 10px; padding: 5px 15px;">Dyslexia</li><li class="elderly" style="box-sizing: border-box; position: relative; align-items: center; display: flex; background-color: rgb(255, 255, 255); border: 2px solid rgb(74, 102, 129); border-radius: 5px; font-size: 1rem; margin-inline-end: 10px; margin-bottom: 10px; padding: 5px 15px;">Elderly</li></ul></div><div class="wcag-criteria-wrapper" style="box-sizing: border-box; position: relative; display: flex; flex-direction: column; margin-top: 1rem;"><strong role="heading" style="box-sizing: border-box; position: relative; font-weight: bolder;">WCAG 2.0, 2.1 &amp; 2.2</strong><ul style="box-sizing: border-box; position: relative; padding: 0px; margin: 0px; list-style-type: none; align-items: center; display: flex;"><li style="box-sizing: border-box; position: relative; align-items: center; display: flex; margin-inline-end: 15px; margin-bottom: 10px;"><span class="level" style="box-sizing: border-box; position: relative; align-items: center; border: 1px solid rgb(74, 102, 129); border-radius: 100%; display: flex; flex: 0 0 40px; font-weight: 500; height: 40px; justify-content: center; margin-inline-end: 10px; width: 40px;">AA</span><span class="message" style="box-sizing: border-box; position: relative;">1.4.3 Contrast (Minimum)</span></li><li style="box-sizing: border-box; position: relative; align-items: center; display: flex; margin-inline-end: 15px; margin-bottom: 10px;"><span class="level" style="box-sizing: border-box; position: relative; align-items: center; border: 1px solid rgb(74, 102, 129); border-radius: 100%; display: flex; flex: 0 0 40px; font-weight: 500; height: 40px; justify-content: center; margin-inline-end: 10px; width: 40px;">AAA</span><span class="message" style="box-sizing: border-box; position: relative;">1.4.6 Contrast (Enhanced)</span></li></ul></div></div>
Index | Elements | Status
-- | -- | --
1 | <button type="submit" class="search-button">Go!</button> | Failed
2 | <a href="/donate/" class="donate-link"> Donate </a> | Failed
3 | <a href="/ebooks/categories.html#history" data-emoji="🏛️">History</a> | Failed
4 | <a href="/ebooks/categories.html#literature" data-emoji="📜">Literature</a> | Failed
5 | <a href="/ebooks/categories.html#science-technology" data-emoji="🔭">Science &amp; Technology</a> | Failed
6 | <a href="/ebooks/categories.html#social-sciences-society" data-emoji="🧑‍🤝‍🧑">Social Sciences &amp; Society</a> | Failed
7 | <a href="/ebooks/categories.html#arts-culture" data-emoji="🎨">Arts &amp; Culture</a> | Failed
8 | <a href="/ebooks/categories.html#religion-philosophy" data-emoji="☯️">Religion &amp; Philosophy</a> | Failed
9 | <a href="/ebooks/categories.html#lifestyle-hobbies" data-emoji="🎯">Lifestyle &amp; Hobbies</a> | Failed
10 | <a href="/ebooks/categories.html#health-medicine" data-emoji="🌿">Health &amp; Medicine</a> | Failed
11 | <a href="/ebooks/categories.html#education-reference" data-emoji="🎓">Education &amp; Reference</a> | Failed
12 | <h1 id="slogan"> Project Gutenberg is a library of over 75,000 free eBooks </h1> | Failed
13 | <h5>Nainen sen tietää by Kaarle Halme</h5> | Failed
14 | <h5>Naimisiin by Kaarle Halme</h5> | Failed
15 | <h5>East of the sun and west of the moon by Theodore Roosevelt and Kermit Roosevelt</h5> | Failed
16 | <h5>Meister Eckart by Hans Lassen Martensen</h5> | Failed
17 | <h5>Récits héroïques by Jules Claretie</h5> | Failed
18 | <h5>L'espionne by Ernest Daudet</h5> | Failed
19 | <h5>From a London garden by Arthur St. John Adcock</h5> | Failed
20 | <h5>Overcoming handicaps by Archer Wallace</h5> | Failed
21 | <h5>The story of Pocahontas and Captain John Smith by E. Boyd Smith</h5> | Failed
22 | <h5>Frankenstein; Or, The Modern Prometheus by Mary Wollstonecraft Shelley</h5> | Failed
23 | <h5>Moby Dick; Or, The Whale by Herman Melville</h5> | Failed
24 | <h5>Romeo and Juliet by William Shakespeare</h5> | Failed
25 | <h5>Pride and Prejudice by Jane Austen</h5> | Failed
26 | <h5>The Strange Case of Dr. Jekyll and Mr. Hyde by Robert Louis Stevenson</h5> | Failed
27 | <h5>Alice's Adventures in Wonderland by Lewis Carroll</h5> | Failed
28 | <h5>The Complete Works of William Shakespeare by William Shakespeare</h5> | Failed
29 | <h5>A Room with a View by E. M. Forster</h5> | Failed
30 | <h5>Middlemarch by George Eliot</h5> | Failed
31 | <h5>The Scarlet Letter by Nathaniel Hawthorne</h5> | Failed
32 | <a href="/about/">About Project Gutenberg</a> | Failed
33 | <a href="/policy/privacy_policy.html">Privacy policy</a> | Failed
34 | <a href="/policy/permission.html" title="Permissions, Licensing and other Common Requests">Permissions</a> | Failed
35 | <a href="/policy/terms_of_use.html">Terms of Use</a> | Failed
36 | <a href="/about/contact_information.html" title="How to contact Project Gutenberg">Contact Us</a> | Failed
37 | <a href="/help/" title="Help, How-To, Procedures, Guidance and Information">Help</a> | Failed
38 | <h5>"Quite wild animals" by Beatrice Curtis Brown</h5> | Failed
39 | <p style="margin: 0; font-family: inherit; font-style:italic; font-size: 16px; line-height: 1.6; color: #333; text-align: center;"> The Project Gutenberg comm...</p> | Failed

</div>
https://freeaccessibilitychecker.skynettechnologies.com/?website=www.gutenberg.org

**Comment by eshellman** [comment]
I think there must be something wrong with the checker; the computed contrast is about 6

**Comment by Rossolson** [comment]
Agreed. It looks like Skynet is flagging the H5 content, which is https://webaim.org/resources/contrastchecker/?fcolor=495057&bcolor=D9EAEF well within the contrast recommendation. Perhaps it's chocking on the CSS stylesheet in some weird way.
